pkgs/tools/misc/fontforge/default.nix

```diff
@@ -31,11 +31,12 @@ stdenv.mkDerivation rec {
     python freetype zlib glib libungif libpng libjpeg libtiff libxml2
   ]
     ++ lib.optionals withGTK [ gtk2 pango ]
-    ++ lib.optionals (withGTK && stdenv.isDarwin) [ Carbon Cocoa ];
+    ++ lib.optionals stdenv.isDarwin [ Carbon Cocoa ];
 
   configureFlags =
     lib.optionals (!withPython) [ "--disable-python-scripting" "--disable-python-extension" ]
-    ++ lib.optional withGTK "--enable-gtk2-use";
+    ++ lib.optional withGTK "--enable-gtk2-use"
+    ++ lib.optional stdenv.isDarwin "--without-x";
```

Comment:
I only get a command line interface. Maybe the problem is that nix'es X libraries don't know how to talk to my system's Xquartz. We can drop this commit if you want, it should still build on hydra.